puzzle.html

```diff
@@ -8,13 +8,16 @@ <h5>
       {{puzzle.name}}<br>
       <button class="btn btn-inverse bb-set-link-button" title="Edit link">Set Puzzle Link</button>
     {{/if}}
+    {{> puzzle_add_tag}}
   </h5>
   {{#with puzzle}}
   {{#if solved}}
-    <b>SOLVED! Answer: {{tag "answer"}}{{#if tag "backsolved"}} (backsolved){{/if}}</b>
+    <b>SOLVED! Answer: {{tag "answer"}}{{#if backsolved}} (backsolved){{/if}}</b>
+  <!--
```

Comment:
Can we add a TODO to remove it at some point in the future (e.g. after MH assuming WOW doesn't blow up)?

Reply:
I was thinking about just leaving it for a while until we can figure out what everything going on here is. I've found a lot of loose ends here and there that appear to just have had the frontend bit deleted and it's pretty confusing. Having this explicitly disabled bit here makes it easier to understand imo. wdyt?